tests/system/test_general.py

```diff
@@ -0,0 +1,52 @@
+from .base import BaseSystemTestCase
+
+
+class WSGIApplicationTester(BaseSystemTestCase):
+    """
+    Tests the WSGI application in general.
+    """
+
+    def test_request_wrong_method(self) -> None:
+        response = self.client.get("/")
+        self.assertEqual(response.status_code, 405)
+
+    def test_request_wrong_media_type(self) -> None:
+        response = self.client.post("/")
+        self.assertEqual(response.status_code, 400)
+        self.assertIn("Wrong media type.", str(response.data))
+
+    def test_request_missing_body(self) -> None:
+        response = self.client.post("/", content_type="application/json")
+        self.assertEqual(response.status_code, 400)
+        self.assertIn("Failed to decode JSON object", str(response.data))
+
+    def test_request_fuzzy_body(self) -> None:
+        response = self.client.post(
+            "/", json={"fuzzy_key_Eeng7pha3a": "fuzzy_value_eez3Ko6quu"},
+        )
+        self.assertEqual(response.status_code, 400)
+        self.assertIn("data must be array", str(response.data))
+
+    def test_request_fuzzy_body_2(self) -> None:
+        response = self.client.post(
+            "/", json=[{"fuzzy_key_Voh8in7aec": "fuzzy_value_phae3iew4W"}],
+        )
+        self.assertEqual(response.status_code, 400)
+        self.assertIn(
+            "data[0] must contain [\\'action\\', \\'data\\'] properties",
+            str(response.data),
+        )
+
+    def test_request_no_existing_action(self) -> None:
+        response = self.client.post(
+            "/", json=[{"action": "fuzzy_action_hamzaeNg4a", "data": [{}]}],
```

Comment:
Why is `/` the action service? How do you differentiate form presenters? This can be done in another PR.

Reply:
Differentiation between actions and presenters is done in the Initialization of the application (depending on the view given [here](https://github.com/OpenSlides/openslides-backend/blob/8842bf1fdd0f1852aeeaae1e14a65d6071802a92/tests/system/util.py#L48). The routes are just `/` for both. I noticed that as well, but didn't have the time yet to look into it, since we said every route should have the prefix `/internal/<service>`, but on the other side, routing is handled by the haproxy anyway. Create an issue if this needs to be changed.

Reply:
This is no problem, if everything is public. I was just curios how this can be actions, if nowhere in this file the word `action` is used (except for some teststrings). EIther there is some magic (but even the file location does not hint that there are actions, not presenters), or some base-class name is not well choosen. Please have a look and make it clear, why here are actions and not presenters. Also: If this is action-specific, this should not be named `test_general` without any hint for actions.

Reply:
Good catch, I somehow even threw away all presenter tests... Well, they are back now and I moved the base classes/tests to the right folders.